charts/portal/README.md

```diff
@@ -289,6 +289,7 @@ This section describes configurable parameters in **values.yaml**, there is also
 | `dispatcher.additionalLabels`        | A list of custom key: value labels                           | `not set`                                                    |
 | `dispatcher.additionalEnv.CONFIG_HTTPS_TLS` | Enabled HTTPS TLS Versions                            | `If not specfied, Portal TLS defaults are enabled` see [Portal TLS Defaults](#portal-tls-defaults)                                                   |
 | `dispatcher.additionalEnv.CONFIG_HTTPS_CIPHER_SUITE` | Enabled HTTPS Cipher Suites                  | `If not specfied, Portal Cipher Suites defaults are enabled` see [Portal Cipher Suites Defaults](#portal-cipher-suites-defaults)                                                   |
+| `dispatcher.additionalEnv.CONFIG_HOST_ALLOWED_DOMAINS` | The &#124; separated allowed domain(s) whitelist. | ``                                                   |
```

Comment:
I suggest `Use &#124; to separate allowed domains.` for the description, and maybe even adding an example: `e.g. mydomain1.com | mydomain2.com`

Reply:
updated.
![image](https://user-images.githubusercontent.com/69872713/191626309-fc1c97e3-c9da-4df7-a830-5ead3636b4cf.png)